sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_models.py

```diff
@@ -326,10 +312,10 @@ class LeaseProperties(BlobLeaseProperties):
     :ivar str duration:
         When a file is leased, specifies whether the lease is of infinite or fixed duration.
     """
-    def __init__(self):
-        self.status = None
-        self.state = None
-        self.duration = None
+    def __init__(self, **kwargs):
+        super(LeaseProperties, self).__init__(
```

Comment:
i dont think this __init__ is needed to just call the super

Reply:
agree!